src/controllers/cart.controller.js

```diff
@@ -0,0 +1,47 @@
+const cartService = require("../services/cart.service");
+
+const createCart = async (req, res) => {
+  try {
+    const { productId } = req.body;
+    const userId = res.locals.payload.id;
+    await cartService.createCart(userId, productId);
+    res.status(201).json({ message: "Item has been put into the cart!" });
+  } catch (err) {
+    res.status(err.statusCode || 400).json({ message: err.message });
+  }
+};
+
+const editCart = async (req, res) => {
+  try {
+    const { productId, quantity } = req.body;
+    const userId = res.locals.payload.id;
+    await cartService.editCart(userId, productId, quantity);
+    res.status(201).json({ message: "Number of item has been edited!" });
+  } catch (err) {
+    res.status(err.statusCode || 400).json({ message: err.message });
+  }
+};
+
+const deleteCart = async (req, res) => {
+  try {
+    const { productId } = req.body;
+    const userId = res.locals.payload.id;
+    await cartService.deleteCart(userId, productId);
+    res.status(201).json({ message: "Item has been deleted from the cart!" });
+  } catch (err) {
+    res.status(err.statusCode || 400).json({ message: err.message });
+  }
+};
+
+const getCart = async (req, res) => {
+  try {
+    const userId = res.locals.payload.id;
+    const products = await cartService.getCart(userId);
+    console.log(products);
+    res.status(201).send(products);
```

Comment:
json() 함수와 send() 함수의 차이를 학습하면 json() 함수가 조금 더 나을 것 같다는 결론에 같이 도달할 것 같습니다.

json() 함수는 해당 함수로 받는 프로퍼티의 data 타입을 체크하지 않고 바로 send() 함수를 호출하게 됩니다.

반면 send() 함수는 해당 함수로 받는 프로퍼티의 데이터 타입을 체크하면서 object일 경우 json() 함수를 다시 호출하고 마지막으로 send() 함수를 호출하게 됩니다.  

```suggestion
    res.status(201).json(products);
```